test/unit/argument_parser/format_parse_validators_test.cpp

```diff
@@ -208,27 +208,40 @@ TEST(validator_test, output_file)
 
     { // single file
 
-        { // empty list of file.
-            seqan3::output_file_validator my_validator{};
-            EXPECT_NO_THROW(my_validator(tmp_name.get_path()));
+        { // file does not exist (& no formats given)
+            std::filesystem::path not_existing_path{tmp_name_2.get_path()};
+            seqan3::output_file_validator my_validator{seqan3::output_file_open_options::open_or_create};
+            EXPECT_NO_THROW(my_validator(not_existing_path));
+            seqan3::output_file_validator my_validator2{seqan3::output_file_open_options::create_new};
+            EXPECT_NO_THROW(my_validator2(not_existing_path));
+            seqan3::output_file_validator my_validator3{}; // default: create_new
+            EXPECT_NO_THROW(my_validator3(not_existing_path));
         }
 
-        { // file does not exist.
+        { // file does exist & overwriting is prohibited
             std::ofstream tmp_file_2(tmp_name_2.get_path());
-            std::filesystem::path does_not_exist{tmp_name_2.get_path()};
-            seqan3::output_file_validator my_validator{formats};
-            EXPECT_THROW(my_validator(does_not_exist), seqan3::validation_error);
+            std::filesystem::path existing_path{tmp_name_2.get_path()};
+            seqan3::output_file_validator my_validator{seqan3::output_file_open_options::create_new, formats};
+            EXPECT_THROW(my_validator(existing_path), seqan3::validation_error);
+        }
+
+        { // file does exist but allow to overwrite it
+            std::ofstream tmp_file_2(tmp_name_2.get_path());
```

Comment:
Can you do the creation above? It is weird and unexpected that we change the property of `tmp_file_2` in the middle of the tests.